source/text.tex

```diff
@@ -12238,8 +12238,8 @@
 constructs an end-of-sequence iterator object,
 which is the only legitimate iterator to be used for the end
 condition. The result of \tcode{operator*} on an end-of-sequence iterator is not
```

Comment:
Most of this prose is just repeating general iterator requirements (you can't dereference the end iterator, dereferencing it returns the iterator's reference type etc).